src/core/backend/core.rs

```diff
@@ -16,7 +17,8 @@ use std::{pin::Pin, sync::Arc};
 pub struct BackendOperation {
     vdisk_id: VDiskId,
     disk_path: Option<DiskPath>,
-    alien: bool, // flag marks data belonging for different node
+    remote_node_name: Option<String>, // save data to alien/<remote_node_name>
```

Comment:
if we have `remote_node_name` do we need `alien` flag?

Reply:
in next branch i want to refact this logic